src/sentry/api/endpoints/organization_auditlogs.py

```diff
@@ -46,10 +46,14 @@ def get(self, request: Request, organization) -> Response:
             else:
                 queryset = queryset.filter(event=query["event"])
 
-        return self.paginate(
+        response = self.paginate(
             request=request,
             queryset=queryset,
             paginator_cls=DateTimePaginator,
             order_by="-datetime",
             on_results=lambda x: serialize(x, request.user),
         )
+        # TODO: Cleanup after frontend is fully moved to version 2
+        if "version" in request.GET:
```

Comment:
Do you care what the value of `version` is? Right now `version=cats` will get the v2 response.